dist/openshift/cincinnati-deployment.yaml

```diff
@@ -187,17 +187,26 @@ objects:
         name: cincinnati
       minReplicas: ${{MIN_REPLICAS}}
       maxReplicas: ${{MAX_REPLICAS}}
+      # Using 'Pods' type, as the metric is summed by pod, so each pod has its own value and 
+      # HPA will automatically average across all pods. We want to scale based on per-pod metrics
       metrics:
         - type: Pods
           pods:
             metric:
-              name: cincinnati_pe_graph_incoming_requests_per_second
-              selector:
-                matchLabels:
-                  uri_path: "/api/upgrades_info/v1/graph"
+              name: cincinnati_request_rate
             target:
               type: AverageValue
-              averageValue: "${{PE_REQ_AVG}}"
+              averageValue: ${{PE_REQ_AVG}}
+  - apiVersion: monitoring.coreos.com/v1
+    kind: PrometheusRule
+    metadata:
+      name: cincinnati-recording-rule
+    spec:
+      groups:
+        - name: cincinnati.rules
+          rules:
+            - record: cincinnati_request_rate
+              expr: sum by (pod) (rate(cincinnati_pe_graph_incoming_requests_total[1m]))
```

Comment:
[Default `scrapeInterval` is 30s][1], so `1m` only gives time for 2 (and very rarely 3?) scrapes.  We'd get a more reliable rate estimate that could handle occasional scrape jitter or failures if we used `2m` or similar.

[1]: https://docs.openshift.com/container-platform/4.16/rest_api/monitoring_apis/prometheus-monitoring-coreos-com-v1.html#spec